addons/mail/static/src/models/composer/composer.js

```diff
@@ -270,6 +272,136 @@ function factory(dependencies) {
             await this.env.bus.trigger('do-action', { action, options });
         }
 
+        parseBody() {
+            const escapedAndCompactContent = escapeAndCompactTextContent(this.textInputContent);
+            let body = escapedAndCompactContent.replace(/&nbsp;/g, ' ').trim();
+            body = this._generateMentionsLinks(body);
+            body = parseAndTransform(body, addLink);
+            body = this._generateEmojisOnHtml(body);
+            return body;
+        }
```

Comment:
Feels like this introduced a lot of duplicate code, should at least use new methods in existing post message (until we remove it in master).